File: backend/src/FaJuan.Api/Domain/Entities/Banner.cs
namespace FaJuan.Api.Domain.Entities;

public class Banner
{
    public long Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public long ImageAssetId { get; set; }
    public string? LinkUrl { get; set; }
    public int Sort { get; set; }
    public bool IsEnabled { get; set; } = true;
    public DateTime CreatedAt { get; set; } = DateTime.Now;
}

File: frontend/src/types/banner.ts
export interface BannerListItemDto {
  id: number
  title: string
  imageAssetId: number
  imageUrl: string
  linkUrl?: string
  sort: number
  isEnabled: boolean
  createdAt: string
}

export interface SaveBannerRequest {
  title: string
  imageAssetId: number
  linkUrl?: string
  sort: number
  isEnabled: boolean
}
